docs/get-started/frameworks/angular.mdx

```diff
@@ -14,7 +14,7 @@ To install Storybook in an existing Angular project, run this command in your pr
 
 <CodeSnippets path="create-command.md" variant="new-users" copyEvent="CreateCommandCopy" />
 
-You can then get started [writing stories](/docs/get-started/whats-a-story/), [running tests](/docs/writing-tests/) and [documenting your components](/docs/writing-docs/). For more control over the installation process, refer to the [installation guide](/docs/install/).
+You can then get started [writing stories](../whats-a-story.mdx), [running tests](../../writing-tests/index.mdx) and [documenting your components](../../writing-docs/index.mdx). For more control over the installation process, refer to the [installation guide](../install.mdx).
```

Comment:
I did find it harder for the 404 detection tool I tried to work with relative paths. I'm also myself not so keen to use relative paths because when we move a file around:
* the links in that file all change
* the links to that file also change, and those can be harder to find and change than a unique absolute path